documentation/modules/cruise-control/con-rebalance-goals-proposals.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-rebalance-goals-proposals-{context}']
+= Rebalance goals and proposals
+
+To rebalance a Kafka cluster, Cruise Control uses _rebalance goals_ to generate _rebalance proposals_. 
+
+[discrete]
+== Rebalance goals
+
+Rebalance goals specify targets for resource distribution and utilization across a Kafka cluster. 
```

Comment:
We are not redistributing resources (we are not moving CPUs and RAM) we are redistributing load.

Reply:
Good point. I've updated to "workload redistribution".